lib/components/DraftailEditor.js

```diff
@@ -243,12 +247,12 @@ class DraftailEditor extends Component<Props, State> {
 
     this.renderSource = this.renderSource.bind(this);
 
-    const { rawContentState } = props;
+    const { rawContentState, readOnly } = props;
 
     this.state = {
       editorState: conversion.createEditorState(rawContentState),
       hasFocus: false,
-      readOnly: false,
+      readOnlyState: readOnly,
```

Comment:
I don’t think it’s a good idea to use the prop to override the local state – I presume that if someone uses the prop then they want complete control over this, and Draftail internally switching the state back to `false` would be an issue. So instead the prop can be used directly in `render()`.

Reply:
Yea it's because it's an uncontrolled component the `DrafttailEditor` I thought it made sense that  the prop would be mirrored into state and not taken into consideration in subsequent renders... but I changed the code so it's always resolved from both places. 👍